nff/io/ase_calcs.py

```diff
@@ -454,13 +468,11 @@ def set(self, **kwargs):
 
         The special keyword 'parameters' can be used to read
         parameters from a file."""
-        changed_params = Calculator.set(self, **kwargs)
```

Comment:
same here? @sauradeep93 this maybe ruff or something

Reply:
ok so I checked this two changes. they are from @xiaochendu. He can comment better

Reply:
Oh boy, I have these deleted lines in my code. Can we revert these?

Reply:
can u suggest how i think ruff did it for him or something

Reply:
😂 I think the easiest way might be to copy these lines back? But strange, I'm not sure why ruff would do something like this.

Reply:
Also @ajhoffman1229 might have better ideas

Reply:
i guess then it's what u thought. He didn't have it or something. let's see lol

Reply:
No idea how these changes came across, because I didn't do them. There could be a difference between my local repo and the GitHub repo. Anyways I copy pasted the stuff as it was before. Please have a look and let me know incase I missed something.
Sorry for the inconvenience

Reply:
Thanks! I just added 2 additional (small) comments!